internal/testrunner/runners/system/servicedeployer/compose.go

```diff
@@ -87,7 +87,7 @@ func (r *DockerComposeServiceDeployer) SetUp(ctxt ServiceContext) (DeployedServi
 		return nil, errors.Wrap(err, "could not create STDOUT file")
 	}
 	service.stdout = outFile
-	ctxt.STDOUT = agentLogsFolder + stdoutFileName
+	service.ctxt.STDOUT = agentLogsFolder + stdoutFileName
```

Comment:
In general we shouldn't modify the context as service property, but rather build the new context first and then assign to the service.

Let's keep it here for now as we're in the middle of debugging the issue.

Reply:
https://github.com/elastic/elastic-package/pull/101